menu.rb

```diff
@@ -0,0 +1,50 @@
+class Menu
+  MENU_OPTIONS = {
+    '1' => :list_books,
+    '2' => :list_people,
+    '3' => :create_person,
+    '4' => :create_book,
+    '5' => :create_rental,
+    '6' => :list_all_rentals,
```

Comment:
- [ ] Kindly fix the following error that appears after listing books or listing people. After listing the data the app should redisplay the menu options so the user can navigate to other features.

![image](https://github.com/Eccb7/school-library/assets/62290545/b8fe08c7-ccbb-4e53-ba8c-33c476bfdf5a)